README.md

```diff
@@ -1,4 +1,16 @@
-# nx-set-shas
+<p style="text-align: center;"><img src=".github/assets/nx.png" 
+width="100%" alt="Nx - Smart, Extensible Build Framework"></p>
+
+<h1 align="center">Set SHAs Action</h2>
+
+> ✨ A Github Action which sets the base and head SHAs required for `nx affected` commands in CI
+
+- [Example Usage](#example-usage)
+- [Configuration Options](#configuration-options)
+- [Background](#background)
+- [License](#license)
+
+> This documentation is for version 2.x.x. If you are using version 1.x.x you also need to include accomplanying [nx-tag-successful-ci-run](https://github.com/nrwl/nx-tag-successful-ci-run) as the last step of your job, since version 1.x.x depends on the existance of git tags that mark successful runs.
```

Comment:
There are a couple of typos in here (`accomplanying`, `existance`) so it needs to be changed anyway, so may I suggest instead of this paragraph we instead simply link to the README.md at the latest v1 tag?